multibranch/pom.xml

```diff
@@ -88,7 +88,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>cloudbees-folder</artifactId>
-            <version>5.2.1</version>
+            <version>5.8-SNAPSHOT</version>
```

Comment:
I understand why you are bumping up the dependency, I am just pointing out that it is not _necessary_ because you could trivially implement `getCategoryId()` here, just leaving a one-line TODO comment to delete the override once the `cloudbees-folder` dependency is updated to 5.8+. The advantage would be that this PR could be merged now, rather than sometime later.